website/content/docs/configuration/index.mdx

```diff
@@ -221,6 +221,10 @@ a negative effect on performance due to the tracking of each lock attempt.
   the `VAULT_EXPERIMENTS` environment variable as a comma-separated list, or via the
   [`-experiment`](/vault/docs/commands/server#experiment) flag.
 
+- `imprecise_lease_role_tracking` `(bool: "false")` - If no role based quota exists for mount, do not keep track of lease roles.
+  Role based quotas include rate limit quotas or a lease count quotas that have a role specified role configured.
+  The purpose of this configuration is to reduce expensive role calculations if role base quotas are not being used.
```

Comment:
How do you feel about something like:

```suggestion
- `imprecise_lease_role_tracking` `(bool: "false")` - Skip pre-quota lease counting for roles if no role-based quota exists for mount. After a role-based quota is enabled for a mount, subsequent leases reset. This applies to both Lease Count and Rate Limit Quotas utilizing `ResolveRoleOperation`.
```

Reply:
I would change the last sentence to avoid the ambiguity of using "this" as a pronoun. Something like "The least count reset affects role-based Lease Count Quotas, but reduces latencies when not using the feature."

Reply:
How about this:
```Skip lease counting by role if there are no role based quotas enabled. Role based quotas include rate limit quotas or a lease count quotas that have a role specified in configuration. When `imprecise_lease_role_tracking` set to true and a new role-based quota is enabled, subsequent lease counts start from 0.  This affects role-based Lease Count Quotas, but reduces latencies when not using role based quotas.
```

Reply:
We're still relying on "this" to tie the ideas together. How about:

> Skip lease counting by role if there are no role based quotas enabled. Role
> based quotas include rate limit quotas or lease count quotas that have a role
> specified in configuration. When `imprecise_lease_role_tracking` is set to `true`
> and a new role-based quota is enabled, subsequent lease counts start from 0.
> Restarting the lease count affects role-based lease count quotas, but reduces
> latencies when not using role based quotas.

Reply:
thanks for all the help! that looks great, and im going to link the docs as well